build_stage_repository

```diff
@@ -39,13 +39,17 @@ def get_rpm_name(rpm):
     return check_output(command, universal_newlines=True)
 
 
-def modulemd_yaml(collection, version):
+def modulemd_yaml(collection, version, foreman_version):
```

Comment:
Can we make this here, and in other places, have a default value of None? To match my question around having a `--foreman-version` option?